lib/jellyfish/room.ex

```diff
@@ -0,0 +1,186 @@
+defmodule Jellyfish.Room do
+  @moduledoc """
+  Module representing room.
+
```

Comment:
```suggestion
```

Reply:
I'm not sure what does that mean, to remove the `moduledoc` (well, it doesn't convey much information)?

Reply:
I just meant to remove that one line